src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -128,20 +128,20 @@ var adyenHelperObj = {
   getCheckoutEnvironment() {
     let returnValue = '';
     switch (AdyenConfigs.getAdyenEnvironment()) {
-      case adyenHelperObj.MODE.TEST:
-        returnValue = adyenConstants.CHECKOUT_ENVIRONMENT_TEST;
+      case constants.MODE.TEST:
+        returnValue = constants.CHECKOUT_ENVIRONMENT_TEST;
         break;
-      case adyenHelperObj.MODE.LIVE:
+      case constants.MODE.LIVE:
         const frontEndRegion = AdyenConfigs.getAdyenFrontendRegion();
         if(frontEndRegion === adyenHelperObj.FRONTEND_REGIONS.US ) {
```

Comment:
I don't think this if statement works. FRONTEND_REGIONS are not defined in `adyenHelperObj`